src/Config.php

```diff
@@ -228,6 +218,22 @@ private static function getTerminusScript()
     }
 
     /**
+     * Ensures that directory paths work in any system
```

Comment:
I have Cygwin installed on a VirtualBox, so let me know if you'd like me to test.